Simplified/NYPLBarcodeScanningViewController.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2012 ZXing authors
```

Comment:
Let's make sure we update the list of licenses in the app to include this license.

Reply:
Let me know if I've attributed that correctly. It's the locally-bundled `software-licenses.html` file